src/docfx/config/EnvironmentVariable.cs

```diff
@@ -19,6 +19,8 @@ public static class EnvironmentVariable
 
         public static string? CorrelationId => GetValue("DOCFX_CORRELATION_ID");
 
+        public static bool RemoveHostName => bool.TryParse(GetValue("DOCFX_REMOVE_HOST_NAME"), out var removeHostName) && removeHostName;
```

Comment:
This is not needed because the environment var is auto populated during config load

Reply:
ok